data/records/lhcb-ntuples-hackathon.json

```diff
@@ -0,0 +1,1022 @@
+[
+  {
+    "abstract": {
+      "description": "<p>Data from proton-proton (pp) collisions collected by the LHCb experiment filtered to produce Ntuples for {insert reason for request}.</p><p>FIXME Here should come a long explanation of inputs, ntuples, etc.</p><p>FIXME If we can attach the <code>info.yaml</code> etc, this is the place wher they would be added."
```

Comment:
RE: If we can attach the code...

I will have to check with DPA but I agree this would be nice. Then we can mention that some information about the tools used to save quantities to the Ntuples are specified in the config files.